src/test/unittest_triangulation.cpp

```diff
@@ -0,0 +1,117 @@
+#include <gtest/gtest.h>
+#include "helpers.h"
+
+using namespace Eigen;
+using namespace xivo;
+
+#include "core.h"
+#include "math.h"
+
+TEST(Triangulation, Normal_Inputs) {
+
+  /*
+  3D point = (2,3,5)
+
+  Translation = 10
+
+  Rotation = 10 degree around y axis
+  */
+
+  Vec2 xc1{2/5,3/5};
+  Vec2 xc2{-8.7447 / 3.5397,3 / 3.5397};
```

Comment:
For all of these examples, I would actually define `Xc1` (a `Vec3`), and `g12`. Then project `Xc1` to get `xc1` and transform it through `g12` to get `Xc2` and then `xc2`. There should be functions in `common/se3.h` that make this easy. That way, test cases are easily differentiated from one another:
- parallax is easily defined as an itty-bitty change in pose
- chierality can be a really big rotation + small translation that moves the point out of the camera's FOV
- Vanishing point can just be a really really large depth + the same translation

Reply:
Addressed in the commit!